states/maine/elections.toml

```diff
@@ -1,9 +1,47 @@
 [20200714]
 date = 2020-07-14
 name = "Congressional Primary"
-original_date = 2020-07-14
+original_date = 2020-06-19
+
+[20200714.register]
+# received_by = ?? # Not specified
```

Comment:
Maybe received_by 21 days before? `There is no cut-off date for registering to vote in person at your town office or city hall. If you want to register to vote by mail or through a voter registration drive, the cut-off date is the close of business on the 21st day before the election.`

Reply:
Bingo ! thanks.